code/datums/components/crafting/tailoring.dm

```diff
@@ -6,6 +6,59 @@
 	time = 5 SECONDS
 	category = CAT_CLOTHING
 
+/datum/crafting_recipe/durathread_robe
+	name = "Durathread Robe"
+	result = /obj/item/clothing/suit/wizrobe/durathread
+	reqs = list(/obj/item/stack/sheet/durathread = 3,
+				/obj/item/stack/sheet/leather = 6)
```

Comment:
this and all your other lists should be reformatted, if you're multilining you want a newline at the start and the end, like this.
Make sure to keep tabs consistent too, only need one.
Adding a , to the end of the last entry is also good practice since it makes expanding lists easier
```suggestion
	reqs = list(
		/obj/item/stack/sheet/durathread = 3,
		/obj/item/stack/sheet/leather = 6,
	)
```